code/README.md

```diff
@@ -1,19 +1,10 @@
-# Technigo React Starter App
+Happy thoughts project.
 
-This app has been generated using `create-react-app`, then cleaned up a little so we have a project with just the elements we need to get started.
+This project was about creating a social media web app where you can post happy thoughts. 
 
-Add components in the `src` folder and use them in `src/app.js` to get started.
+It´s created with a joint API for the whole boot camp so that all the thoughts being displayed att everyones pages.
+Its build in react with fetch from a external API, and with a seperate for the counting of likes. 
 
-## Usage
+This project was pretty straight forward with clear goals and an already made up design. 
 
-1. On GitHub, click the 'use this template' button to create a new repo on your account
-1. Clone the new repo to your computer
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
-
-Alternatively, if you don't want a new git repo on your account;
-
-1. Download & extract the [latest release](https://github.com/Technigo/react-starter/releases/latest) from GitHub
-1. Open the directory in the terminal: `cd /path/to/react-starter`
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
+If I had more time I would add animation to newly posted thought and a charachter counter. 
```

Comment:
Think you can use the below solution for a character counter just change the state to your naming for thoughts: 

   ```
 //Condition to change text color to red when there are only 10 characters left to reach max
        style={{ color: newThought.length > 130 ? "red" : "black" }}>
        {140 - newThought.length} characters left
```